.github/workflows/ci.yaml

```
@@ -0,0 +1,31 @@
+name: CI
```

Adding version name for future version control documentation.
```suggestion
name: CI
version: 1.0
```